libraries/AP_HAL/HAL.h

```diff
@@ -19,7 +19,9 @@ class AP_HAL::HAL {
         AP_HAL::UARTDriver* _uartC, // telem1
         AP_HAL::UARTDriver* _uartD, // telem2
         AP_HAL::UARTDriver* _uartE, // 2nd GPS
-        AP_HAL::I2CDriver*  _i2c,
+        AP_HAL::I2CDriver*  _i2c0,
+        AP_HAL::I2CDriver*  _i2c1,
+        AP_HAL::I2CDriver*  _i2c2,
```

Comment:
Much of the noise in the resulting diff is due to this conversion. We can improve it on top: let drivers ask for a particular bus and use their own private _i2c variables.